### DISCUSSION TITLE
INPUT ERROR: could not open genomeFastaFile

### DISCUSSION THREAD

**Discussion by leylasahverdi** [alexdobin/STAR#1850] [Q&A] [answered]
/home/leylasahverdii/STAR-2.7.10b/source/STAR --runThreadN 7 --runMode genomeGenerate --genomeDir /home/leylasahverdii/index --genomeFastaFiles /home/leylasahverdii/index/reference/hg38.fa " "
        STAR version: 2.7.10b   compiled: 2023-05-09T09:45:42+00:00 :/home/leylasahverdii/STAR-2.7.10b/source
May 09 12:49:57 ..... started STAR run
May 09 12:49:57 ... starting to generate Genome files

EXITING because of INPUT ERROR: could not open genomeFastaFile:  

May 09 12:50:12 ...... FATAL ERROR, exiting
./index.sh: line 7: --sjdbGTFfile: command not found

hi, can someone help me with this error, i checked my script many times, nothing  looking wrong

**Comment by alexdobin** [comment, marked as answer]
Something went wrong with the formatting of the command line, there are extra quote characters at the end and the remainder of the command line (`--sjdbGTFfile` ...) was cut off into a new line.

**Reply by leylasahverdi** [reply]
Thanks a lot for reply, i have solved problem by typing script directly on command line ( before i did it by running executable file )
